### ISSUE TITLE
 Delete Posts by Post Type - Add tests

### ISSUE THREAD

**Issue by sudar** [sudar/bulk-delete#250]
## Tests
- [x] Create helper function to create posts in a custom post type
- [x] Modify existing helper function to retrieve posts in a custom post type
- [x] Add tests to test deleting posts from a single post type (default post type)
- [x] Add tests to test deleting posts from two post types (default post type)
- [x] Add tests to test deleting posts from a single post type (custom post type)
- [x] Add tests to test deleting posts from two post types (custom post type)
- [x] Add tests to test deleting posts from a built-in type and a custom post type
- [x] Add tests to test deleting posts permanently ( at least 2 test cases)
- [x] Add tests to test deleting posts that are older than x days (use at least two terms for this)
- [x] Add tests to test deleting posts that are posted within the last x days (use  at least two terms for this)
- [x] Create more than 100 posts and try to delete them in batches. Test at least 2 batches.
- [ ] Test that scheduling works by checking if a cron job is scheduled.

Once this task is done, the helper functions should be moved to the tests library repo.

While sending this task for code review create a new issue for moving the helper functions to the the test library repo
